test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # Running Parity Full Test Sute
 
-FEATURES="json-tests ipc"
```

Comment:
better disable the test
since you risk leaving main binary contain compile-time errors while all tests are passing